css/style.css

```diff
@@ -282,11 +289,45 @@ footer ul li {
   list-style: none;
   margin: 0 26px 75px;
 }
-footer p {
-  font-size: 18px;
+.copyright {
+  font-size: 0.692rem;
   color: #fff;
   font-weight: 600;
 }
 
+/******************************************
+/* SCREEN BREAKS
+/*******************************************/
+
+@media screen and (max-width: 1000px) {
+  #benefits { width: 100%; padding: 60px 40px; }
+}
 
+@media screen and (max-width: 768px) {
+  .container { width: 100%; margin: 0 auto; }
+  header .logo { margin-left: 20px; }
+  .main-nav { display: none; margin-right: 20px; }
+  .mbl-nav {
+    display: block;
+    padding: 0 0 0 90%;
+    color: #fff;
+    font-size: 1.5rem;
+  }
+  #benefits { width: 100%; padding: 60px 40px; }
+  #benefits p, #benefits ul { width: 100%; }
+  #benefits ul { margin: 20px 15px 0 45px; }
+  #main { width: 100%; margin: 20px 30px; }
+  #posts { width: 90%; }
+  #posts img { width: 100%; }
+  #sidebar { width: 95%; }
+}
 
+@media screen and (max-width: 480px) {
+  #benefits { width: 100%; padding: 30px 0 30px 20px; }
+  #benefits p, #benefits ul { width: 100%; }
```

Comment:
Rather than setting these to `100%` here, its probably better if you just change the `width` value defined above the media query to a `max-width`. That way the width with never grow beyond the `max-width`, but it can also shrink to as small as it needs to.

This also applies to `blockquote`